plugin/helpers_kv.go

```diff
@@ -220,3 +222,25 @@ func (p *HelpersImpl) KVListWithOptions(options ...KVListOption) ([]string, erro
 
 	return ret, nil
 }
+
+func (p *HelpersImpl) KVModify(key string, f func(value []byte) ([]byte, error)) error {
+	err := p.ensureServerVersion("5.2.0")
+	if err != nil {
+		return err
+	}
+
+	v, appErr := p.API.KVGet(key)
+	if appErr != nil {
+		return appErr
+	}
+
+	modifiedValue, err := f(v)
+	if err != nil {
+		return err
+	}
+	_, appErr = p.API.KVSetWithOptions(key, modifiedValue, model.PluginKVSetOptions{})
```

Comment:
I believe we want to set `PluginKVSetOptions.Atomic = true` and set the old value as well, since we want this operation to be atomic.